### PR TITLE
anchor platform: update for next rc

### PR DESCRIPTION
**Pull request by JakeUrban**

### CODE REVIEW

File: docs/anchoring-assets/anchor-platform/installation.mdx [hunk 1]
```diff
@@ -29,15 +29,15 @@ Let's create a minimal compose file to get started.
...
-  image: stellar/anchor-platform:2.0.0-rc.2
-  command: --sep-server
-  ports:
-    - "8080:8080"
-  env_file:
-    - ./dev.env
-  volumes:
-    - ./config:/home
+  platform-server:
+    image: stellar/anchor-platform:2.0.0-rc.2
```
Comment (Ifropc) [inline docs/anchoring-assets/anchor-platform/installation.mdx:33]: rc.3

**Comment by Ifropc** [comment]
We should update documentation regarding JWT and the requested asset.
Now, it's part of the JWT data
https://github.com/stellar/java-stellar-anchor-sdk/pull/887
Also, it's not adviced to fetch if from db requested_amount anymore

**Comment by JakeUrban** [comment]
Do you think you could take that on?

**Comment by Ifropc** [comment]
Closed in favor of https://github.com/stellar/stellar-docs/pull/156

**Pull request closed without merging**